libs/widget-lib/src/themeUtils.ts

```diff
@@ -0,0 +1,11 @@
+import { CowSwapTheme, CowSwapWidgetPalette } from './types'
+
+export function isCowSwapWidgetPallet(
```

Comment:
I think you use `pallet` in several points, when you mean `pallete` 
this is a pallet https://m.media-amazon.com/images/I/61VsHVLs3GL._AC_SL1500_.jpg

Reply:
Ooops, typo, thanks!